contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java

```diff
@@ -447,6 +445,25 @@ private void checkIfQuotaExceeded(Long diskQuotaSizeInKB,
         }
     }
 
+    private boolean retryIsFile(FileSystem fs, Path rootPath) {
```

Comment:
should it be just called isFile ?

retryIsFile , seems to be conveying wrong information.